### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
New lines update
You can check it @jordimas @darccio @ferranrego @MiquelAdell @johnnydement @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -298,7 +298,7 @@ msgstr "Afegeix text alternatiu"
 
 #: src/view/com/composer/GifAltText.tsx:175
 msgid "Add ALT text"
-msgstr ""
+msgstr "Afegeix ALT"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:301]: The English string was a typo that was fixed in #3934 to become `Add alt text` (the message catalogs haven't been updated with it yet). So I suggest you might want to change this to follow the translation below, something like `Afegeix text alternatiu`?

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -5724,7 +5724,7 @@ msgstr "Aquesta apel·lació s'enviarà a <0>{0}</0>."
 
 #: src/screens/Messages/Conversation/MessageListError.tsx:26
 msgid "This chat was disconnected due to a network error."
-msgstr ""
+msgstr Aquest xat s'ha desconnectat degut a un problema de xarxa.""
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:5727]: There's a `"` missing at the beginning here and an extra `"` at the end.

Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Thanks! your corrections are so useful!

**Review by pfrazee — APPROVED** [review]
To make sure this gets into the release, I'm considering this sufficiently reviewed. Thanks all!

**Pull request merged**